content/articles/transfer-account-ownership.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: Transfer Account Ownership
+excerpt: How to transfer ownership of the account to another DNSimple user.
+categories:
+- Account
+---
+
+# Transferring Account Ownership
+
+Generally, you'll transfer account ownership by [changing the user email](/articles/changing-email#changing-the-user-email) to that of the individual who's taking over the account, who then [requests a password reset](/articles/forgot-password/). If you run into any issues with this, please [reach out to us](https://dnsimple.com/contact) so that we can help further.
+
+## A note about accounts
+
+When you initially sign up as a user in our system, we create an account bound to that user. We call this the Primary Account, and the email address is the same as the username. If you change the email address for that user, the account email address will also be updated. That way customers with a single account only have to update their email in one place. Follow the instructions in [Changing User Email](/articles/changing-email#changing-the-user-email) if this is the case for you.
```

Comment:
I feel like there is a missed opportunity here to expand on the account email vs the user email. What do you think?